GoogleForCreators/web-stories-wp
# KSES: don’t strip page attachment URLs

## Summary

Got an error report saying that the link was dropped when using page attachments.

This PR fixes that.

## Relevant Technical Choices

Added a unit test.

Since there's also AMP sanitization happening later, we could think about extending our existing e2e test for author users.

## To-do



## User-facing changes



## Testing Instructions

1. Log in as author
1. Add page attachment to story
1. View story on frontend
1. Page attachment link should work correctly

---



Fixes #5728
